specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1304 @@
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_GetById": {
+            "$ref": "./examples/PrivateClouds_GetById.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds": {
+      "get": {
+        "operationId": "PrivateClouds_ListInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "list private clouds in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponseList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_ListInResourceGroup": {
+            "$ref": "./examples/PrivateClouds_ListInResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}": {
+      "get": {
+        "operationId": "PrivateClouds_GetByIdInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "get a private cloud in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_GetByIdInResourceGroup": {
+            "$ref": "./examples/PrivateClouds_GetByIdInResourceGroup.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "PrivateClouds_CreateOrUpdate",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "create a private cloud",
```

Comment:
[nitpick] "Create or update a private cloud"

Reply:
I'm going to resolve these as I go through them and save locally. I'll push a commit in an hour or so.